io.elementary.mail.json

```diff
@@ -101,10 +89,10 @@
                     ],
                     "sources": [
                         {
-                            "type": "git",
-                            "url": "https://github.com/libical/libical.git",
-                            "tag": "v3.0.9"
-                        }
+                            "type": "archive",
+                            "url": "https://github.com/libical/libical/releases/download/v3.0.16/libical-3.0.16.tar.gz",
+                            "sha256": "b44705dd71ca4538c86fb16248483ab4b48978524fb1da5097bd76aa2e0f0c33"
+                         }
```

Comment:
Any reason to prefer the archive over the git repo? if so, wouldn't be better to use archive for all dependencies?

Also, there's a extra space before the close brace.

Reply:
Flatpak is way more efficient when dealing with archives than git repositories (it only has to look at if the file is newer on the server before downloading it)